opalog/opalog.peg

```diff
@@ -69,10 +82,6 @@ Number <- '-'? Integer ( '.' DecimalDigit+ )? Exponent? {
     return t, err
```

Comment:
If JSON doesn't define a limit on the size of a number, then to be truly JSON compliant, we need to handle BigNums.  Does Go's ParseFloat do that?  If not, I'd imagine we should leave a TODO.